torch/ao/quantization/utils.py

```diff
@@ -370,3 +373,105 @@ def has_no_children_ignoring_parametrizations(module):
         return len(module._modules) == 1 and 'parametrizations' in module._modules
     else:
         return False
+
+def _get_path_of_module(root, submodule):
```

Comment:
for all of these util functions, would be nice to add types to inputs/outputs, and provide an example in the docblock of example input and example output.  Not everyone will know that a path or a fqn means.